src/Files.App/Views/SettingsPages/Advanced.xaml

```diff
@@ -234,6 +234,9 @@
 							<ListView.ItemTemplate>
 								<DataTemplate xmlns:vm="using:Files.Backend.ViewModels.FileTags" x:DataType="vm:ListedTagViewModel">
 									<Grid ColumnSpacing="16">
+										<Grid.KeyboardAccelerators>
+											<KeyboardAccelerator Key="Escape" Invoked="KeyboardAccelerator_Invoked" />
```

Comment:
Should this be moved to the text box?

Reply:
This way you can also cancel color change pressing `Esc`

Reply:
I don't think that's needed, users mainly expect to be able to cancel from the text box.

Reply:
Yes, but moving that to `TextBox` would close settings if the user hits `Esc` when editing color

Reply:
Ignore what I said, I just realized this is only in the list view item template. I thought it was on the whole page.

Reply:
I believe the user would expect to exit `Edit Mode` and then exit settings pressing another time `Esc`

Reply:
> Ignore what I said, I just realized this is only in the list view item template. I thought it was on the whole page.

Ok